### PR TITLE
{bio}[foss/2021a] bcbio-gff v0.6.7 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by manifestoso**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.7-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+bcbiogffroot = 'lib/python%(pyshortver)s/site-packages'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.7-foss-2021a.eb:32]: @robqiao as with the other version, please use the `local_` prefix for these variables cf. https://easybuild.readthedocs.io/en/latest/Easyconfig-files-local-variables.html

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.7-foss-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['6e6f70639149612272a3b298a93ac50bba6f9ecece934f2a0ea86d4abde975da']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.7-foss-2021a.eb:25]: also missing `Biopython`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2633.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/dfb0be839d5e1bfb963d468b6b26c027 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/ea73f16aca35762de70e7b4fb0b5bf4c for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @manifestoso!

**Pull request merged**